src/GraphType.jl

```diff
@@ -147,10 +147,21 @@ mutable struct GraphData
 
         # equivalence classes (at the beginning each state represents just itself)
         eq_vn(v0, p0) = (v0 == spp[p0] ? nothing : pvers[p0][v0])
-        eq_classes = Dict(pkgs[p0] => Dict(eq_vn(v0,p0) => Set([eq_vn(v0,p0)]) for v0 = 1:spp[p0]) for p0 = 1:np)
+
+        # Hot code
+        eq_classes = Dict{UUID,Dict{InstState,Set{InstState}}}()
+        for p0 = 1:np
+            d = Dict{InstState, Set{InstState}}()
+            for v0 = 1:spp[p0]
+                let p0 = p0
+                    d[eq_vn(v0,p0)] = Set([eq_vn(v0,p0)])
+                end
+            end
+            eq_classes[pkgs[p0]] = d
+        end
 
         # the resolution log is actually initialized below
-        rlog = ResolveLog(uuid_to_name, verbose)
+        rlog = ResolveLog(uuid_to_name, false)
```

Comment:
?

Reply:
Sorry, had to remove keyword arguments to use `code_warntype`, forgot to put this back.